practice-02/src/index.html

```diff
@@ -138,34 +138,34 @@
             <div class="box-logo">
                 <div class="list-logo">
                     <button class="btn logo-item">
-                        <img src="//cdn.tgdd.vn/Brand/1/logo-iphone-220x48.png" width="30" class="logo" alt="Logo iPhone">
+                        <img src="//cdn.tgdd.vn/Brand/1/logo-iphone-220x48.png" width="30" name="iPhone" class="logo" alt="Logo iPhone">
                     </button>
                     <button class="btn logo-item">
-                        <img src="//cdn.tgdd.vn/Brand/1/samsungnew-220x48-1.png" width="30" class="logo" alt="Logo Samsung">
+                        <img src="//cdn.tgdd.vn/Brand/1/samsungnew-220x48-1.png" width="30" name="Samsung" class="logo" alt="Logo Samsung">
```

Comment:
name attr should be lowercase. Update for ALL CASES

Reply:
Edited and added .lowCase() method to filter capitalized product names